codegen/test_data/endpoints/bar_bar_method_argnotstruct.gogen

```diff
@@ -30,14 +30,12 @@ import (
 
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
+	module "github.com/uber/zanzibar/.tmp_gen/endpoints/bar/module"
```

Comment:
lol the linter change

Reply:
I hope this import ordering/grouping thing is final.